.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea or enhancement for the tool.
+title: ''
+labels: 'enhancement'
```

Comment:
I'm not sure what is different between `enhancement` and `cmty:feature-request`.
Which should I use?

Reply:
enhancement is good